curriculum/Chapter-2-Agile-Team-Collaboration.md

```diff
@@ -0,0 +1,140 @@
+
+A Sprint is initiated by a Sprint planning. Sprint planning lays out the work to be done within a Sprint, and the goal of these planning sessions are to answer three questions:
+
+- Why is this Sprint valuable?
+- What can be Done this Sprint?
+- How will the chosen work get done?
+
+Scrum responsibilities can vary or even be shared among roles within a balanced team, but as a developer you are primarily accountable for: creating a plan for the Sprint and maintaining the Sprint Backlog, instilling quality by in iterations adhering to acceptance criteria, adapting planss toward the sprint goal, and holding other devlopers accountable.
+
+This backlog of work is continually reviewed and prioritzed by devs within the "daily scrum". Developers can select whatever structure and techniques they want, as long as their Daily Scrum focuses on progress toward the Sprint Goal and produces an actionable plan for the next day of work.
+Daily scrums improve communications, promote quick decision-making, and consequently eliminate the need for other meetings.
+
+After the completion of each Sprint, the scrum team holds a Sprint retrospective to assess the work done. The team discusses what went well during the Sprint, problems encountered, and how these problems were (or can be) solved. The outcome of a retrospective are action items to improve the effectiveness of the Sprint, which can be incoroprated as soon as the next Sprint cycle. The purpose of these meetings are to increase the efficiency and quality of the Sprint cycle, reinforcing Agile principles like iterative and continuous improvement.
+
+It is important not to conflate the topics of Agile and scrum. Agile set of guiding principles aimed at efficiently delviering value to the end user, and scrum is a framework to apply these principles and ship software to prod. While scrum is an important framework that provides value to balanced teams, it is important to ensure your team is operating in a manner that can be considered "Agile".
+
+## Types of Work
+
+Agile development categorizes work into distinct types, each serving a unique role in delivering user value efficiently. Understanding these categories helps teams prioritize tasks, manage workloads, and maintain a user-focused approach throughout the product lifecycle.
+
+### Discovery and Framing
+
+This initial phase involves researching and defining what to build next, focusing on understanding user needs, market trends, and potential impacts on the product. Effective discovery and framing ensure that the team invests in work that maximizes user and business value.
+
+### Feature Development
+
+Features are new capabilities or enhancements added to a product to meet user needs or improve user experience. Focusing on features allows teams to directly address user requests or market demands, ensuring the product evolves in a way that adds real value.
```

Comment:
We like to break down work into various categories. This changes depending on the team you're on, but its generally:

- Spike: desc
- Bug Fix: desc
- Feature: desc
- Chore: desc